site/src/styles/global.css

```
@@ -4,80 +4,119 @@
+  --chart-4: oklch(0.55 0.012 58);
+  --chart-5: oklch(0.68 0.14 76);
+  --sidebar: oklch(0.94 0.015 78);
+  --sidebar-foreground: oklch(0.22 0.02 45);
+  --sidebar-primary: oklch(0.58 0.16 50);
+  --sidebar-primary-foreground: oklch(1 0 0);
+  --sidebar-accent: oklch(0.55 0.12 66);
+  --sidebar-accent-foreground: oklch(1 0 0);
+  --sidebar-border: oklch(0.88 0.03 78);
+  --sidebar-ring: oklch(0.58 0.16 50);
+  --font-sans: 'Inter', 'Oxanium', system-ui, sans-serif;
+  --font-serif: Merriweather, serif;
```

Stylelint (17.5.0)

[error] Expected "Merriweather" to be "merriweather" (value-keyword-case)

(value-keyword-case)